tests/console/docker.pm

```diff
@@ -108,6 +108,16 @@ sub run {
     $output = script_output('docker container run tw:saved curl -I google.de');
     die("network is not working inside of the container tw:saved") unless ($output =~ m{Location: http://www\.google\.de/});
 
+    # docker run support --init parameter
+    assert_script_run 'docker run --rm --init opensuse/tumbleweed ps --no-headers -xo "pid args" | grep "1 /dev/init"';
+
+    # if man command is available
```

Comment:
I would remove the comment **if man command is available**. The if statement should be clear enough. (at least, it is clear for me)

Reply:
I would put in the comment something like: **check manual pages**